src/status_im/events.cljs

```diff
@@ -193,8 +192,19 @@
 (fx/defn on-will-focus
   {:events [:screens/on-will-focus]
    :interceptors [anon-metrics/interceptor]}
-  [cofx view-id]
+  [{:keys [db] :as cofx} view-id]
   (fx/merge cofx
+            (cond
+              (= :chat view-id)
+              {::async-storage/set! {:chat-id (get-in cofx [:db :current-chat-id])}
+               :db (assoc db :screens/was-focused-once? true)}
```

Comment:
could you elaborate on was-focused-once? ?

Reply:
Yes, sure. On non development build `:screens/on-will-focus` is dispatched before we have enough time to handle `:chat-id` from async storage. Thus if do not ignore the first `:screens/on-will-focus` `:chat-id` is set to `nil`.